build.gradle

```diff
@@ -18,7 +18,7 @@ sourceCompatibility = JavaVersion.VERSION_1_7
 targetCompatibility = JavaVersion.VERSION_1_7
 
 group = 'com.google.cloud.tools'
-version = '0.1.2-SNAPSHOT'
+version = '0.2.0-SNAPSHOT'
```

Comment:
@meltsufin @loosebazooka 

Why are we still on 0.2.0 if these are 1.0 RC ready?